pallet/account-migration/src/lib.rs

```diff
@@ -43,8 +43,6 @@
 #![cfg_attr(not(feature = "std"), no_std)]
 #![deny(missing_docs)]
 
-// TODO: update weight
```

Comment:
Why remove this? The recommended practice is adding benchmarked weight even for unsigned transactions. Although the unsigned transactions are free, they are part of the block, and occupied the block space and limited by the whole block weight.

Reply:
https://github.com/darwinia-network/darwinia-2.0/issues/100